### PR TITLE
Redux in React Components

### PR DESCRIPTION
**Pull request by IsmaelMastronardi**
On this Branch, I added the: 

- Wrap the App with the Provider.
- configured store.
- added initial state to bookSlice.
- Display the book on a React Component.
- Added a button to add and remove a book.

In the previous branch I had already added some of these changes, like adding the functionalities to the remove and add buttons, and displaying the array of books to check that everything works, please check those changes out.

### CODE REVIEW

File: src/components/book.js
Comment (Efiamotu-1) [inline src/components/book.js:1]: - [ ] Currently when i add multiple books for example I add two books and I try to remove one of the books I added, the two previously added books gets removed, this shouldn't be the functionality of the application, please kindly work on the handleDelete to delete the exact book specified 👍

Comment (Efiamotu-1) [inline src/components/book.js:1]: - [ ] You are currently disabling eslint in this file, kindly fix the linters rather than disabling eslint 👍

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @IsmaelMastronardi , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ademibowale — APPROVED** [review]
Hi @IsmaelMastronardi 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️
- Creates 1 reducer, checkStatus✔️
-  Creates 2 reducers, `addBook` and `removeBook`✔️
- Uses `configureStore` to create a reducer✔️
- Added initial state to bookSlice✔️
- Display the book on a React Component✔️
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good use of Github flow ✔️






- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**